src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.android.cs

```diff
@@ -22,18 +27,124 @@ public class MauiMediaElement : CoordinatorLayout
 	public MauiMediaElement(Context context, StyledPlayerView playerView) : base(context)
 	{
 		this.playerView = playerView;
+		this.context = context;
 
-		// Create a RelativeLayout for sizing the video
-		RelativeLayout relativeLayout = new(context)
+		playerView.FullscreenButtonClick += PlayerView_FullscreenButtonClick;
+		this.playerView.SetBackgroundColor(Android.Graphics.Color.Black);
+
+		AddView(playerView);
+	}
+
+	/// <summary>
+	/// Allows the video to enter or exist a full screen mode
+	/// </summary>
+	/// <param name="sender"></param>
+	/// <param name="e"></param>
+	void PlayerView_FullscreenButtonClick(object? sender, StyledPlayerView.FullscreenButtonClickEventArgs e)
+	{
+		// Ensure there is a player view
+		if (this.playerView is null)
+		{
+			return;
+		}
+
+		// Ensure current activity exists
+		var currentActivity = Platform.CurrentActivity;
+		if (currentActivity is null)
```

Comment:
we could reduce the number of lines on those checks... Something like

```suggestion
		if (Platform.CurrentActivity is not Activity currentActivity
		    || currentActivity.Window is not Window currentWindow)
		    {
		        return;
		    }
```

Reply:
Very nice way to reduce the lines. I learned something new today! Thanks @pictos for the recommendation. It has been applied. Marked as resolved.